CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## [0.8.5] -   2022-09-08
+
+### Changed
+
+- build: python 3.10 support only.
+- dependencies updates:
+   -  bump types-chardet from 5.0.1 to 5.0.4
+   -  bump s3fs from 2022.5.0 to 2022.7.1
+   -  bump geopandas from 0.11.0 to 0.11.1
+   -  bump pre-commit from 2.19.0 to 2.20.0
+   -  bump pytest-sugar from 0.9.4 to 0.9.5
+   -  bump types-pyyaml from 6.0.9 to 6.0.11
+   -  bump urllib3 from 1.26.9 to 1.26.11
+   -  bump pytest-mock from 3.8.1 to 3.8.2
+   -  bump types-chardet from 4.0.4 to 5.0.1
+   -  bump pytest-mock from 3.7.0 to 3.8.1
+   -  bump mkdocs-material from 8.3.6 to 8.3.9
+   -  bump types-python-slugify from 5.0.4 to 6.1.0
+   -  bump black from 22.3.0 to 22.6.0
+   -  bump aiobotocore from 2.3.3 to 2.3.4
+   -  bump types-paramiko from 2.11.1 to 2.11.2
+   -  bump types-dataclasses from 0.6.5 to 0.6.6
+   -  bump chardet from 4.0.0 to 5.0.0
+   -  bump types-pyyaml from 6.0.8 to 6.0.9
+   -  bump types-paramiko from 2.11.0 to 2.11.1
+   -  bump geopandas from 0.10.2 to 0.11.0
+   -  bump mkdocs-material from 8.3.3 to 8.3.6
+   -  bump types-paramiko from 2.10.0 to 2.11.0
+   -  bump certifi from 2022.5.18.1 to 2022.6.15
+   -  bump actions/setup-python from 3 to 4
+   -  bump jq from 1.2.3 to 1.3.0
+   -  bump docker from 5.0.3 to 6.0.0
```

Comment:
I don't think this is relevant for end users. The only change that could impact them is `chardet` bumped from `^4.0.0` to `>=4,<6`, meaning they could end up with chardet 5 if they did not pin it. However, if they're using it , it should be pinned by them anyway so I don't think it is relevant to mention it either